src/libm/CMakeLists.txt

```diff
@@ -205,6 +205,7 @@ foreach(SIMD ${SLEEF_HEADER_LIST})
   )
 endforeach()
 
+file(MAKE_DIRECTORY ${sleef_BINARY_DIR}/include)
```

Comment:
Lgtm, but we probably need the same change in `src/quad/CMakeLists.txt` line 131.